ui/js/controllers/jobs.js

```diff
@@ -205,6 +205,19 @@ treeherderApp.controller('ResultSetCtrl', [
             });
         };
 
+        $scope.fillAllJobs = function(revision) {
+            if (!window.confirm('This will trigger all jobs for revision ' + revision + '!\n\nAre you sure?')) {
```

Comment:
I would rather say "This will trigger all missing jobs for this push.\n\nDo you want to proceed?"